controllers/homeRoutes.js

```diff
@@ -69,4 +73,34 @@ router.get('/gifts/:id', async (req, res) => {
   }
 });
 
+// Use withAuth middleware to prevent access to route
```

Comment:
Cailin, I forgot to comment out this code.  Sorry!  just go ahead and comment it out prior to merging after the approvals.  Thanks!